src/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -62,8 +65,11 @@ internal T WithHttpManager(IHttpManager httpManager)
         }
 
         /// <summary>
+        /// Sets the logging callback. For details see https://aka.ms/msal-net-logging
         /// </summary>
-        /// <returns></returns>
+        /// <returns>The builder to chain the .With methods</returns>
+        /// <exception cref="InvalidOperationException"/> is thrown if the loggingCallback
```

Comment:
Do we want to continue doing this?  We had talked about doing exceptions only at final validation time.  This was put in because we can only have one callback so if you set it, and then set it again, I wanted to let people know.  But it's somewhat against our general philosophy.  Would be good to talk on this one a bit to finalize.